client/client_test.go

```diff
@@ -6,14 +6,18 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/statechannels/go-nitro/client/engine/chainservice"
 	"github.com/statechannels/go-nitro/client/engine/messageservice"
+	"github.com/statechannels/go-nitro/client/engine/store"
 	"github.com/statechannels/go-nitro/types"
 )
 
 func TestNew(t *testing.T) {
-	a := types.Address(common.HexToAddress(`a`))
+
+	pkA := common.Hex2Bytes("c417e8a75ebe2bfe16fe108e1e04802c324974eef6ea2cc3d55194fa38677b5e")
```

Comment:
```suggestion
	skA := common.Hex2Bytes("c417e8a75ebe2bfe16fe108e1e04802c324974eef6ea2cc3d55194fa38677b5e")
```

Reply:
`private` / `public` are both p. `secret` / `public` is probably better when abbreviating.